pkgs/development/tools/turso-cli/default.nix

```
@@ -23,6 +26,13 @@ buildGo121Module rec {
     echo "v${version}" > internal/cmd/version.txt
   '';
 
+  postInstall = ''
```

```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

Cross-compilation works for this package, and it can be kept working like this